actioncable/Rakefile

```diff
@@ -1,15 +1,14 @@
 require 'rake/testtask'
 require 'pathname'
-require 'sprockets'
-require 'coffee-script'
 require 'action_cable'
+require 'blade'
 
 dir = File.dirname(__FILE__)
 
 task :default => :test
 
 task :package => "assets:compile"
-task "package:clean" => "assets:clean"
+task "package:clean"
```

Comment:
Removed `package:clean` in af777bf6ae67c8ec3ef200415d238614fefef68e and updated `assets:compile` to ensure a clean path in db84cb439a2c85cdace279451caad4bf40f7eb33.